pkg/backend/remote/remote.go

```diff
@@ -51,6 +51,12 @@ type Remote struct {
 
 func (r *Remote) Close() error {
 	logrus.Infof("Closing: %s", r.name)
+
+	// Close the dataconn client to avoid orphaning goroutines.
+	if dataconnClient, ok := r.ReaderWriterUnmapperAt.(*dataconn.Client); ok {
+		dataconnClient.Close()
+	}
+
```

Comment:
Need to backport to v1.5.x and v1.6.x?

Reply:
@derekbit and @PhanLe1010,

My current plan is to just backport this specific fix to v1.7.x, v1.6.x, and v1.5.x. However, I can also see backporting the whole functionality since there are no external parameters, etc. to deal with (it is self-contained in the engine). WDYT?

Reply:
I would prefer not to have the new feature (last replica logger timeout) in the stable release of 1.6.3 to be safe and only backport the bug fix https://github.com/longhorn/longhorn-engine/pull/1176/commits/109aaf7f2a1df5a0532c5f9900abcb95e5fbcb24 to 1.6.3

WDYT? @derekbit @ejweber

Reply:
Sure. I'm fine with it. Let's make it only in v.1.7.1+.
Feel free to remove `backport/v1.6.3` label. Thank you.

Reply:
I am fine with it as well. Thanks for the input @PhanLe1010.